reana_client/cli/files.py

```diff
@@ -208,8 +215,18 @@ def download_files(ctx, workflow, filenames, output_directory, access_token):
 @add_workflow_option
 @add_access_token_options
 @click.pass_context
-def upload_files(ctx, workflow, filenames, access_token):
-    """Upload files and directories to workflow workspace."""
+def upload_files(ctx, workflow, filenames, access_token):  # noqa: D301
+    """Upload files and directories to workspace.
+
+    The `upload` command permits to upload workflow input files and
```

Comment:
This command permits...  use the same format as for the download command.  Or switch both to this form, i.e. put The `download` command permits... in the other section.